src/pages/jetton/dataRow/token/Token.tsx

```diff
@@ -57,12 +57,12 @@ export const Token = () => {
           <LoadingContainer loading={jettonLoading} loaderWidth="150px">
             {description && (
               <Box sx={{ maxWidth: 300, maxHeight: 60 }}>
-                {description.length > 100 ? (
+                {description.length > 80 ? (
```

Comment:
Seems verbose - we can render a single component and in the text property add the condition which decides whether to render `...`